yarn.lock

```diff
@@ -11613,7 +11613,7 @@ rc-table@~7.26.0:
     rc-util "^5.22.5"
     shallowequal "^1.1.0"
 
-rc-tabs@^12.2.2, rc-tabs@~12.2.0:
+rc-tabs@~12.2.0:
```

Comment:
`yarn install` produced this change (not for the first time). I think, it needs to be there.